packages/rum-recorder/src/domain/segmentCollection/deflateWorker.d.ts

```diff
@@ -24,10 +24,11 @@ export type DeflateWorkerAction =
 export type DeflateWorkerResponse =
   | {
       id: number
-      size: number
+      compressedSize: number
     }
   | {
       id: number
       result: Uint8Array
+      rawSize: number
```

Comment:
what about using `uncompressed` instead of `raw`?
it could more explicit

Reply:
I am okay with both but I would like to settle on a name because I also have to modify the intake of the event platform

Reply:
Following discussion, keeping `raw`